ManageIQ/manageiq
# Adds UI to create/update/delete OpenStack cloud tenants

This PR adds a UI to create/update/delete OpenStack cloud tenants. 

**Steps for Testing/QA**

Create
1. Go to Clouds > Tenants and select "Create Cloud Tenant" under "Configuration".
2. Fill out the cloud tenant creation form and submit.
3. You will be redirected to the list of cloud tenants.  Your new cloud tenant should appear there.
